client/blocks/inline-help/admin-sections.js

```diff
@@ -0,0 +1,424 @@
+/**
+ * External dependencies
+ */
+import { intersection, words, memoize } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import { getLocaleSlug } from 'lib/i18n-utils';
+
+/**
+ * Returns admin section items with site-based urls.
+ *
+ * @param {string} siteSlug - The current site slug.
+ * @returns {Array}           An array of admin sections with site-specific URLs.
+ */
+export const adminSections = memoize( ( siteSlug ) => [
+	{
+		title: translate( 'Add a new domain' ),
+		description: translate(
+			'Set up your domain whether it’s registered with WordPress.com or elsewhere.'
+		),
+		link: `/domains/add/${ siteSlug }`,
+		synonyms: [ 'domain' ],
+		icon: 'domains',
+	},
+	{
+		title: translate( 'Manage my domain settings' ),
+		link: `/domains/manage/${ siteSlug }`,
+		icon: 'domains',
+	},
+	{
+		title: translate( 'Change my site address' ),
+		link: `/domains/manage/${ siteSlug }/edit/${ siteSlug }`,
+		synonyms: [ 'domain' ],
+		icon: 'domains',
+	},
+	{
+		title: translate( 'Add a site redirect' ),
+		description: translate( 'Redirect your site to another domain.' ),
+		link: `/domains/add/site-redirect/${ siteSlug }`,
+		synonyms: [ 'domain', 'forward' ],
+		icon: 'domains',
+	},
+	{
+		title: translate( 'Change my password' ),
+		link: '/me/security',
+		synonyms: [ 'update' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( "Change my site's theme" ),
+		link: `/themes/${ siteSlug }`,
+		synonyms: [ 'switch', 'design' ],
+		icon: 'customize',
+	},
+	{
+		title: translate( "Customize my site's theme" ),
+		link: `/customize/${ siteSlug }`,
+		synonyms: [ 'color', 'font', 'design', 'css', 'widgets' ],
+		icon: 'customize',
+	},
+	{
+		title: translate( 'Change my homepage' ),
+		link: `/customize/${ siteSlug }`,
+		synonyms: [ 'home', 'homepage' ],
+		icon: 'customize',
+	},
+	{
+		title: translate( 'Edit my menu' ),
+		link: `/customize/${ siteSlug }`,
+		synonyms: [ 'menu' ],
+		icon: 'customize',
+	},
+	{
+		title: translate( 'Set a site logo' ),
+		link: `/customize/${ siteSlug }`,
```

Comment:
Themes have to support site logos. Should we add this one only if the current theme supports them?

Deep link:
```suggestion
		link: `/customize/${ siteSlug }?autofocus[section]=title_tagline`,
```